hebiros_advanced_examples/src/omnibase/example_omnibase_odometry.cpp

```diff
@@ -31,6 +31,39 @@ void feedback_callback(sensor_msgs::JointState data) {
   feedback_init = true;
 }
 
+//function to update the odometry pose values
+void updatePose(nav_msgs::Odometry& odom, double dx_global, double dy_global, geometry_msgs::Quaternion& odom_quat){
```

Comment:
Pretty minor, but I'd prefer the references that you aren't changing in these files to be `const`:

`const geometry_msgs::Quaternion& odom_quat`

This prevents you from accidentally modifying them within the function.